core/types/src/account.rs

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Mobilecoin Account and Subaddress objects
```

Comment:
:upside_down_face: (and other places in this PR) 
it looks like it's usually title case *MobileCoin*
```suggestion
//! MobileCoin Account and Subaddress objects
```